templates/content/GeoBlazor.Template.WebApp/GeoBlazor.Template.WebApp/Components/Layout/MainLayout.razor

```diff
@@ -0,0 +1,23 @@
+﻿@inherits LayoutComponentBase
```

Comment:
I'm confused why we have this `Components` folder outside of the two project folders?